test-specifications/utility.md

```diff
@@ -106,19 +106,18 @@ Method used to generate a Hedera Key.
 
 #### Input Parameters
```

Comment:
I am confused what happens if `type` or `fromKey` is not provided, I understand that random key type should be returned then, but like what would be the use case for that?

Reply:
Understood. 

> then the developer won't need to fill in those params but they'll get a usable key

I am for deterministic way of testing in TCK, meaning that in this case there should be no randomness, only things specified by developer.

Would like to have opinions from other team members as well 🙂

Reply:
I agree with Rob on the fact that this can be there for developer convenience most of the times. But Nikita also has a point here because it makes sense to have a function called which does not depend on randomness when used in TCK scope. If a scenario exists in which this randomness can affect us we would have to remove this(So maybe it would be less risky to not have it). 

(Example)
The developer gets used to that function providing him with a working key and gets lazy in specification. Tries to create TCK specifications for `EthereumTransaction` and signs with a random key(not ECDSA).

So if there is no risk to begin with maybe we would be better off.

Reply:
If we want to remove the randomness that's fine, I mostly just wanted something where we didn't have to specify a type every time we call `generateKey` if it doesn't matter what the key is. Maybe we can just default to an `ed25519PrivateKey` if a type isn't specified?

Reply:
Yeah, that was something that also came to my mind if you want to have a default value.

Reply:
I would suggest to remove all the randomness and keep up with the deterministic way of testing. In my opinion  everything in the TCK should be set explicitly.

Reply:
I would go with setting the type explicitly every time. I know it's not convenient but it's way clearer and straightforward, for example you can easily read whether it's ECDSA or ED25519, without having to check the default type.

Reply:
If that's what everyone prefers we can go that way, I'll add it to this

Reply:
@rwalworth that's a valid point you've raised. Just keep in mind we are four people reviewing TCK PRs and sometimes it's possible to miss some details or comments. We will be aware of this in the future!